test/runtests.jl

```diff
@@ -1,7 +1,12 @@
-# The next two lines added as a workaround to
-# https://github.com/JuliaLang/Pkg.jl/issues/3628 (Julia 1.6):
 import Pkg
-Pkg.add(name="Statistics", version=VERSION)
+
+if Base.VERSION >= v"1.10-"
```

Comment:
Thanks for the help here!!

Help me understand why this fixes the issue identified at #574, which is a Julia 1.11.2 failure, as far as I understand it.

Reply:
Ok. Weird. 1.11.2 tests are passing. I'm going to merge and anyway.